pages/index.tsx

```diff
@@ -1,11 +1,141 @@
 import React from 'react';
 import GlobalHeader from '@jetbrains/kotlin-web-site-ui/out/components/header';
 import searchConfig from '../search-config.json';
+import { CtaBlock } from '@jetbrains/kotlin-web-site-ui/out/components/cta-block-v2';
+import { Button } from '@rescui/button';
+import { useTS } from '@jetbrains/kotlin-web-site-ui/out/components/breakpoints';
+
+import YoutubePlayer from '@jetbrains/kotlin-web-site-ui/out/components/youtube-player';
+import '@jetbrains/kotlin-web-site-ui/out/components/layout';
+
+import { KotlinUsageHighlights } from '../blocks/main/kotlin-usage-highlights/kotlin-usage-highlights';
+import GlobalFooter from '@jetbrains/kotlin-web-site-ui/out/components/footer';
+import { ThemeProvider } from '@rescui/ui-contexts';
+
+import GradleLogo from '../public/images/companies/gradle.svg';
+import CordaLogo from '../public/images/companies/corda.svg';
+import EvernoteLogo from '../public/images/companies/evernote.svg';
+import CourseraLogo from '../public/images/companies/coursera.svg';
+import SpringLogo from '../public/images/companies/spring.svg';
+import AtlassianLogo from '../public/images/companies/atlassian.svg';
+import MultiplatformPreviewImage from '../public/images/index/multiplatform-preview.svg';
+
+import { InfoBlock } from '../blocks/main/info-block/info-block';
+import { DividerLine } from '../blocks/main/divider-line/divider-line';
+
+import Link from 'next/link';
+
+import styles from './index.module.css';
+
+const kotlinUsageHighlightsCases = [
```

Comment:
I'm now not sure about keeping all the data on this page instead of putting it into the components. Seems like this page is getting too big

Reply:
The better option is to put it in a YAML file, but I don't know how to refer to the images. Probably by name? Indeed, I can match an image with the company by name, and this contract can be mentioned in Readme.